packages/fx-core/src/core/middleware/envInfoLoader.ts

```diff
@@ -136,6 +153,30 @@ export async function loadSolutionContext(
   return ok(solutionContext);
 }
 
+export async function loadSolutionContextWithoutEnv(
```

Comment:
Can we reuse the `loadSolutionContext`? For example, add a parameter to skip loading env config.

Reply:
Discussed offline with @dooriya. For local debug, it needs to read from default environment's profile file for upgrading `programmingLanguage`, etc. So just reuse the `loadSolutionContext` to use default env